src/pages/clients/edit/components/Contacts.tsx

```diff
@@ -73,7 +73,7 @@ export function Contacts(props: Props) {
         <div key={index} className="pb-4 mb-4 border-b">
           <Element leftSide={t('first_name')}>
             <InputField
-              id="first_name"
+              id={`first_nam_${index}`}
```

Comment:
If we have more than one contact assigned to a particular client, we will get an error in the console that our elements do not have unique IDs in the DOM. It first appeared when I opened the PassowordConfirmation modal.

Reply:
A subtle change, but `first_nam` is a typo.

Reply:
Resolved!